python/paddle/distributed/auto_parallel/static/engine.py

```diff
@@ -1081,8 +1081,17 @@ def _initialize(self, mode, init_parameters=True):
                 if scope_var and scope_var.get_tensor()._is_initialized():
                     continue
                 uninitialized.append(var)
-            if uninitialized:
-                prune_startup_prog = dist_startup_prog._prune(uninitialized)
+            # Make sure the number of communication operators is consistent
+            commu_ops = []
```

Comment:
commu -> comm, plz fix it in the next pr

Reply:
> commu -> comm, plz fix it in the next pr

Get it! Thanks!